app/uk/gov/hmrc/agentsubscriptionfrontend/repository/ChainedSessionDetailsRepository.scala

```diff
@@ -18,26 +18,26 @@ package uk.gov.hmrc.agentsubscriptionfrontend.repository
 
 import java.util.UUID
 
-import javax.inject.{Inject, Named, Singleton}
+import javax.inject.{Inject, Singleton}
 import org.joda.time.DateTime
 import play.api.libs.json.Json
 import play.modules.reactivemongo.ReactiveMongoComponent
 import reactivemongo.api.indexes.{Index, IndexType}
 import reactivemongo.bson.{BSONDocument, BSONObjectID}
 import uk.gov.hmrc.agentsubscriptionfrontend.config.AppConfig
-import uk.gov.hmrc.agentsubscriptionfrontend.models.KnownFactsResult
-import uk.gov.hmrc.agentsubscriptionfrontend.repository.StashedKnownFactsResult.StashedKnownFactsResultId
+import uk.gov.hmrc.agentsubscriptionfrontend.models.ChainedSessionDetails
+import uk.gov.hmrc.agentsubscriptionfrontend.repository.StashedChainedSessionDetails.StashedChainnedSessionId
 import uk.gov.hmrc.mongo.ReactiveRepository
 import uk.gov.hmrc.mongo.json.ReactiveMongoFormats
 
 import scala.concurrent.{ExecutionContext, Future}
 
 @Singleton
-class KnownFactsResultMongoRepository @Inject()(appConfig: AppConfig, mongoComponent: ReactiveMongoComponent)
-    extends ReactiveRepository[StashedKnownFactsResult, BSONObjectID](
-      "agent-known-facts-results",
```

Comment:
(collection not used any more)if this can be cleaned up easily, lets do it

Reply:
Future tech debt/discussion